utils/oscap_docker_python/get_cve_input.py

```diff
@@ -84,11 +84,23 @@ def _fetch_single(self, dist):
             seconds_epoch = (remote_dt - epoch).total_seconds()
             utime(dest_file, (seconds_epoch, seconds_epoch))
         except KeyError:
-            stderr.write("Response header of HTTP doesn't contain" \
-                  "\"last-modified\" field. Cannot determine version" \
-                  " of remote file \"{0}\"".format(dist_url))
+            self._print_no_last_modified_warning(dist_url)
```

Comment:
Don't we also want to check for `self.DEBUG` here? Or even check for it inside the method.